bluebrain/repo-bluebrain/packages/touchdetector/package.py

```diff
@@ -55,8 +56,8 @@ class Touchdetector(CMakePackage):
     depends_on('range-v3@:0.10', when='@5.3.3:')
     depends_on('libsonata@0.1.9:', when='@5.6.0:')
     depends_on('nlohmann-json', when='@5.3.3:')
-    depends_on('intel-oneapi-tbb', when='@develop')
-    depends_on('caliper@master+mpi', when='+caliper@develop')
+    depends_on('intel-oneapi-tbb', when='@5.7.0:')
+    depends_on('caliper@master+mpi', when='+caliper@5.7.0:')
     depends_on('benchmark', when='+benchmark@develop')
```

Comment:
Does this mean I can't build a release with benchmarks, or sanitizers?

Reply:
We can fix that… depending if one sees these as required for releases or not

Reply:
One more missing on line 104.